.travis.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           - set -e
           - sbt -warn setupPublishCore generateBuildCharacterPropertiesFile headerCheck publishLocal
           - STARR=$(sed -n 's/^maven\.version\.number=//p' buildcharacter.properties) && echo $STARR
-          - sbt -Dstarr.version=$STARR -warn setupValidateTest test:compile info testOsgi
+          - sbt -Dstarr.version=$STARR -warn setupValidateTest compile
```

Comment:
testOsgi I kept in this job because it was failing for me in CI.  So if that's not the case then the following can be deleted:

* the whole of this line
* the whole of the line above it (set STARR)
* the `testOsgi` task key
* the `osgiTests` list can be prepended to `remainingTests`

But that's just clean up on an already great CI improvement, nice work!!

Reply:
I've pushed an additional commit with your suggestions — hopefully CI will like it.

Reply:
This line and the one above can and should be deleted, again...